### PR TITLE
Add default skills

### PR DESCRIPTION
**Pull request by NeonDaniel**
#270

### CODE REVIEW

File: requirements/skills-essential.txt [hunk 1]
```diff
@@ -1,3 +1,9 @@
 ovos-skill-volume~=0.0.1
 ovos-skill-fallback-unknown~=0.0.2
 ovos-skill-stop~=0.2
+neon-skill-alerts~=1.2
+ovos-skill-personal
+ovos-skill-naptime
+ovos-skill-date-time>=0.2.2a1
```
Comment (ChanceNCounter) [inline requirements/skills-essential.txt:7]: I believe this particular specification will cause pip to pull any published prerelease, not just those of the pinned version

**Review by ChanceNCounter — COMMENTED** [review]
I think I should transfer the Dismissal skill to OVOS (should have done a while ago) and ship it with essential, but I don’t think that needs to block 0.0.7

File: requirements/skills-essential.txt (hunk 1 above)
Comment (NeonDaniel) [reply requirements/skills-essential.txt:7]: afaik any `a` in the dependencies will make it so pip considers alpha dependencies.. i.e. `package~=0.1,>=0.0.2a1` would expand to `pip install package >=0.1.0,<1.0.0,>=0.0.2a1` which means all alphas are added to candidate versions

Comment (ChanceNCounter) [reply requirements/skills-essential.txt:7]: Indeed. What I'm concerned about is that `0.0.3a1` will also be captured. If that's what we want, so be it. I've only recently started to get my head around [pip and PEP 440](https://github.com/pypa/pip/issues/7579)

Comment (JarbasAl) [reply requirements/skills-essential.txt:7]: next one, this allows our community to test latest dev branch and report issues before we declare skills stable

**Pull request merged**